### ISSUE TITLE
boxAfterHeader style is no longer needed in SearchPage.jsx

### ISSUE THREAD

**Issue by cindyledev** [Seneca-CDOT/telescope#881]
**What happened**:
After #792 was merged, it added a boxAfterHeader style to add spacing between the Header and the SearchBar component. This is no longer needed as it's not a problem anymore.

![Annotation 2020-03-26 152824](https://user-images.githubusercontent.com/36822198/77688838-160d8e80-6f77-11ea-9ab1-32ca5bafa9fa.png)

Please remove the following in `telescope/src/frontend/src/components/SearchPage/SearchPage.jsx`
- the boxAfterHeader style inside `const useStyles = makeStyles(() => ({`
``` 
boxAfterHeader: {
    minHeight: '12em',
    display: 'flex',
  },
```
- `<Box className={classes.boxAfterHeader}></Box>`

Bonus: if you feel that the SearchBar needs some amount of spacing, modify the `Header.jsx` so that every page will have the same amount of spacing underneath (for consistency). Please be kind, I don't want @agarcia-caicedo to get mad if someone broke the Header...

**What should have happened**:
![Annotation 2020-03-26 152824_](https://user-images.githubusercontent.com/36822198/77689026-64bb2880-6f77-11ea-979c-d7a1dd455c2d.png)

**How to reproduce it (as precise as possible)**:
Visit `https://dev.telescope.cdot.systems/search`

**Anything else we need to know?**:

**Environment**:

- OS: All
- Browser: All

**Comment by humphd** [comment]
I'm suggesting this as a good issue for @JackyTea to tackle next.

**Comment by JackyTea** [comment]
@humphd Haven't setup my dev environment for Telescope yet, but would be happy to tackle this once I do.

**Comment by humphd** [comment]
@JackyTea start by reading through https://github.com/Seneca-CDOT/telescope/blob/master/docs/environment-setup.md and getting your computer setup to work on this code.  Happy to answer any questions you have here or in Slack.

**Comment by neilong31** [comment]
Hopefully it's alright if I work on this! I made a PR that should fix this issue.

**Comment by JackyTea** [comment]
Sorry forgot about this PR. Was trying to get the dev environment to work a while back but never got around to it. Thanks for taking care of it @neilong31.
